src/panels/developer-tools/info/developer-tools-info.ts

```diff
@@ -31,12 +31,18 @@ class HaPanelDevInfo extends LitElement {
 
     const nonDefaultLinkText =
       localStorage.defaultPage === OPT_IN_PANEL && OPT_IN_PANEL === "states"
-        ? "Go to the Lovelace UI"
-        : "Go to the states UI";
+        ? this.hass.localize("ui.panel.developer-tools.tabs.info.lovelace_ui")
+        : this.hass.localize("ui.panel.developer-tools.tabs.info.states_ui");
 
     const defaultPageText = `${
-      localStorage.defaultPage === OPT_IN_PANEL ? "Remove" : "Set"
-    } ${OPT_IN_PANEL} as default page on this device`;
+      localStorage.defaultPage === OPT_IN_PANEL
+        ? this.hass.localize("ui.panel.developer-tools.tabs.info.remove")
+        : this.hass.localize("ui.panel.developer-tools.tabs.info.set")
```

Comment:
Add these as variable in the `default_ui` translation

Reply:
Sure thing!